### PR TITLE
project_key: add test context key

### PR DESCRIPTION
**Pull request by ernestotejeda**
Cc @Tecnativa

### CODE REVIEW

File: project_key/tests/test_common.py [hunk 1]
```diff
@@ -9,8 +9,9 @@ class TestCommon(TransactionCase):
     def setUp(self):
         super(TestCommon, self).setUp()
 
-        self.Project = self.env['project.project']
-        self.Task = self.env['project.task']
+        test_ctx = dict(test_project_key=True)
```
Comment (pedrobaeza) [inline project_key/tests/test_common.py]: Don't do it this way, as you are replacing full context dictionary passing a dictionary argument to next lines.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
The required attribute is being force through a post_init_hook:

https://github.com/OCA/project/blob/12.0/project_key/hooks.py#L9

Remove that line and put the `required` attribute on views.

**Comment by ernestotejeda** [comment]
Changes done

**Review by pedrobaeza — APPROVED** [review]
Fast-tracking this for solving compatibility problems in the rest of the modules tests. What a pain!

**Pull request merged**